static/scss/partials/main.scss

```diff
@@ -849,8 +838,7 @@ input:focus::placeholder {
 }
 
 .relay-email-address-label:hover {
-  background: $color-grey-20;
-  width: 192px;
+  background: $color-grey-10;
```

Comment:
Opted for a lighter grey, it looks closer to the input field on Figma (lemme know if you agree!)